atomic_reactor/util.py

```diff
@@ -661,18 +667,23 @@ def default(self):
         """
         return self.v2_list or self.oci or self.v2 or self.v1
 
+    def __getattr__(self, attr):
+        return self.get(attr, None)
```

Comment:
Should this raise `AttributeError` if `attr not in self`?

Reply:
It shouldn't, the previous version have set other digest versions to `None` by default, so this is what it should return if digest was not set for this version

Reply:
Maybe `AttributeError` if `attr not in self.content_type` then? Otherwise `digest.defalut` (e.g. a typo) will evaluate to None instead of raising an exception.

Reply:
Good idea. Added a test for that